tipue_search/README.md

```diff
@@ -52,16 +49,24 @@ Tipue Search serializes the generated HTML into JSON. Format of JSON is as follo
             "title": "Review of the book Lorem Ipsum"
         }
     ]
-}
+};
 ```
 
-JSON is written to file `tipuesearch_content.json` which is created in the root of `output` directory.
+JS variable is written to file `tipuesearch_content.js` which is created in the root of `output` directory.
 
 How to use
 ==========
 
-To utilize JSON Search mode, your theme needs to have Tipue Search properly configured in it. [Official documentation](http://www.tipue.com/search/docs/#json) has the required details.
+Your theme needs to have Tipue Search properly configured in it. [Official documentation](http://www.tipue.com/search/help/) has the required details.
+
+In addition to the instructions from Tipue, the following has to be added in `pelicanconf.py`.
+
+```python
+PLUGIN_PATH = 'plugins'
+PLUGINS = ['tipue_search']
+DIRECT_TEMPLATES = ['index', 'tags', 'categories', 'authors', 'archives', 'search']```
+
+Furthermore, the generated JS variable has to be sourced in the relevant html pages.
 
-Pelican [Elegant Theme](https://github.com/talha131/pelican-elegant) and [Plumage
```

Comment:
@nkralpy Can you please restore reference to the themes that have tipue working?